File: Practice/src/coding_basic_training/CreateArray1.java
package coding_basic_training;

import java.util.stream.IntStream;

public class CreateArray1 {
//	배열 만들기1{
	public int[] solution(int n, int k) {
		return IntStream.rangeClosed(1, n / k).map(i -> i * k).toArray();
	}
}

File: Practice/src/coding_basic_training/CreateArray1.py
def solution(n, k):
    return list(map(lambda i: i * k, range(1, int(n / k) + 1)))
    # return [i * k for i in range(1, int(n / k) + 1)]
    # return [i for i in range(k, n + 1, k)]
